### ISSUE TITLE
Write-Progress: Length cannot be less than zero. (Parameter 'length')

### ISSUE THREAD

**Issue by dingmeng-xue** [PowerShell/PowerShell#15383]
## Steps to reproduce

The exception can be found from PowerShell-7.2.0-preview.5-win-x64 but no issue is from PowerShell-7.2.0-preview.3-win-x64

PowerShell-7.2.0-preview.5-win-x64
```powershell
Write-Progress -Activity "PUT /subscriptions/000/providers/Microsoft.Authorization/policydefinitions/allowed-locations3?api-version=2019-09-01"
```
Error is printed out
```
Write-Progress: Length cannot be less than zero. (Parameter 'length')
```

PowerShell-7.2.0-preview.3-win-x64 works fine without error.
```powershell
Write-Progress -Activity "PUT /subscriptions/000/providers/Microsoft.Authorization/policydefinitions/allowed-locations3?api-version=2019-09-01"
```

**Comment by iSazonov** [comment]
It seems a dup #15251 and fix is in #15264

**Comment by kasini3000** [comment]
I can reproduce this problem with PowerShell-7.2.0-preview.5-win-x64. Delete 2 characters and the problem disappears.
